packages/fedify/src/codegen/type.ts

```diff
@@ -155,7 +155,7 @@ const scalarTypes: Record<string, ScalarType> = {
     encoder(v) {
       return `{
         "@value": ${v}.toString(),
-        "@language": ${v}.language.compact(),
+        "@language": ${v}.locale.language,
```

Comment:
You should use `baseName` here instead of `language`, because it loses other information like region or script for non-trivial locale code, e.g., `ko-KR`, `zh-Hant-CN`:

```ts
> const locale = new Intl.Locale("ko-KR");
undefined
> locale.language
"ko"
> locale.baseName
"ko-KR"
```